Header header
geometry_msgs/Vector3 force #computed applied forces
geometry_msgs/Vector3 torque #computed applied torques
float64[12] nozzle_openings 
